docs/docs/tuning-your-model.mdx

```diff
@@ -103,6 +103,11 @@ pipeline:
   - name: RegexFeaturizer
   - name: LexicalSyntacticFeaturizer
   - name: CountVectorsFeaturizer
+    # Analyzer to use, either 'word' (default), 'char', or 'char_wb'
```

Comment:
I think rather than having comments explaining the arguments. It might be clearer to have a single comment per `CountVectorsFeaturizer` explaining simply what it is used for e.g. "Counts whole words".

Reply:
Good idea. I'll make that change when I get a chance.